sktm/db.py

```diff
@@ -64,9 +64,8 @@ def createdb(self, db):
                 CREATE TABLE pendingpatches(
                   id INTEGER PRIMARY KEY,
                   pdate TEXT,
-                  patchsource_id INTEGER,
-                  timestamp INTEGER,
-                  FOREIGN KEY(patchsource_id) REFERENCES patchsource(id)
+                  baseurl TEXT,
```

Comment:
We need to add `PRIMARY KEY(baseurl, id)` next, in a separate PR, to fix the bug mentioned in `set_patchset_pending` description.

Reply:
Shouldn't be hard to add here :+1: